docs/topics/api/reviews.rst

```diff
@@ -133,4 +133,37 @@ This endpoint allows you to reply to an existing user review.
 .. http:post:: /api/v3/addons/addon/(int:id|string:slug|string:guid)/reviews/(int:id)/reply/
 
     :<json string body: The text of the reply (required).
-    :<json string|null: The title of the reply.
+    :<json string|null title: The title of the reply.
+
+
+----
+Flag
+----
+
+.. review-flag:
+
+This endpoint allows you to flag an existing user review, to let an editor know
+that something is wrong with it.
+
+ .. note::
+     Requires authentication and a user account different from the one that
+     posted the review.
+
+.. http:post:: /api/v3/addons/addon/(int:id|string:slug|string:guid)/reviews/(int:id)/flag/
+
+    :<json string flag: A :ref:`constant<review-flag-constants>` describing the reason behind the flagging.
+    :<json string|null note: A note to explain further the reason behind the flagging.
```

Comment:
We should maybe specify whether or not the note is required, e.g in case of "other"